docs/configuration.adoc

```diff
@@ -87,6 +87,7 @@ Options for Uptane.
 | `key_source`              | `"file"`     | Where to read the device's private key from. Options: `"file"`, `"pkcs11"`.
 | `key_type`                | `"RSA2048"`  | Type of cryptographic keys to use. Options: `"ED25519"`, `"RSA2048"`, `"RSA3072"` or `"RSA4096"`.
 | `secondary_configs_dir`   | `""`         | Directory containing individual secondary json configuration files. Example here: link:{aktualizr-github-url}/config/secondary/virtualsec.json[]
+| `secondary_config_file`   | `""`         | A path to a json file containing configuration of Secondary ECU(s) for Primary. link:../config/posix_secondary_config.json[Example is here]. It's a new way to configure secondaries and is replacement of `secondary_configs_dir`. The later is going to be removed once the virtual secondary implementation is fully ported to the new approach.
```

Comment:
- "for Primary"  -> "registered on the primary"?
- the link format `link:{aktualizr-github-url}` is also supposed to be more robust and works even if the file is out of the repo.

Reply:
> * the link format `link:{aktualizr-github-url}` is also supposed to be more robust and works even if the file is out of the repo.
ok.

Reply:
> * "for Primary"  -> "registered on the primary"?

Not sure, the IP secondaries details of which are specified in the config, are not registered at the moment of initial aktualizr start. By "for Primary" I meant that this conf file is intended for Primary. Maybe we can rephrase it as "A path to a json file containing configuration of Secondary ECU(s) to be registered at/on/in/with Primary." Not sure what is the correct preposition here :).

Reply:
Almost all of the those prepositions work. "With" might be best.

Reply:
ok. Will update accordingly